simplified-ui-splash/src/main/java/org/nypl/simplified/ui/splash/MailtoWebViewClient.java

```diff
@@ -0,0 +1,48 @@
+package org.nypl.simplified.ui.splash;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.net.MailTo;
+import android.net.Uri;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+
+import java.lang.ref.WeakReference;
+
+/**
+ * Used to handle mailto: links in the eula.html
+ */
+public class MailtoWebViewClient extends WebViewClient {
+    private final WeakReference<Activity> mActivityRef;
+
+    public MailtoWebViewClient(Activity activity) {
+        mActivityRef = new WeakReference<>(activity);
+    }
+
+    @Override
+    public boolean shouldOverrideUrlLoading(WebView view, String url) {
+        if (url.startsWith(MailTo.MAILTO_SCHEME)) {
+            final Activity activity = mActivityRef.get();
+            if (activity != null) {
+                MailTo mt = MailTo.parse(url);
+                Intent i = newEmailIntent(activity, mt.getTo(), mt.getSubject());
+                if (i.resolveActivity(activity.getPackageManager()) != null) {
+                    activity.startActivity(i);
+                }
+                return true;
+            }
+        } else {
+            return false;
+        }
+        return true;
```

Comment:
This return will never execute so you can remove it.

Reply:
@twaddington which one? the `return false;`?

Reply:
Ah, I didn't see the case where activity could be null. We talked in Slack so I'll resolve this.